<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>382</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Resource/edit_Img.png</normaloff>:/Resource/edit_Img.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>57</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>背景图：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>398</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="m_svgPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="m_selectBackFileBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>选择背景</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>顶图：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>398</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLineEdit" name="m_svgTopPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="m_selectTopBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>选择顶图</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_loadFileBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>加载</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>55</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QScrollArea" name="m_svgEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>246</width>
             <height>281</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QListWidget" name="m_imgFileList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>23</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="m_openNewBack"/>
    <addaction name="m_openNewTop"/>
    <addaction name="m_save"/>
    <addaction name="m_openFolder"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="m_help"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑类型</string>
    </property>
    <addaction name="m_forbidEdit"/>
    <addaction name="m_editPolygon"/>
    <addaction name="m_editText"/>
    <addaction name="m_editMile"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_save"/>
   <addaction name="m_forbidEdit"/>
   <addaction name="m_editPolygon"/>
   <addaction name="m_editText"/>
   <addaction name="m_editMile"/>
   <addaction name="m_editLine"/>
   <addaction name="m_saveTopImg"/>
   <addaction name="separator"/>
   <addaction name="m_handlerImag"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <action name="m_refresh">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/refresh.png</normaloff>:/Resource/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新((F5)</string>
   </property>
  </action>
  <action name="m_save">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/save.png</normaloff>:/Resource/save.png</iconset>
   </property>
   <property name="text">
    <string>保存 (Ctrl+S)</string>
   </property>
  </action>
  <action name="m_openNewBack">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/openBack.png</normaloff>:/Resource/openBack.png</iconset>
   </property>
   <property name="text">
    <string>打开新背景图</string>
   </property>
  </action>
  <action name="m_openNewTop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/openTop.png</normaloff>:/Resource/openTop.png</iconset>
   </property>
   <property name="text">
    <string>打开新顶图</string>
   </property>
  </action>
  <action name="m_help">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/help.png</normaloff>:/Resource/help.png</iconset>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="m_editPolygon">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/editPngState.png</normaloff>:/Resource/editPngState.png</iconset>
   </property>
   <property name="text">
    <string>编辑图形</string>
   </property>
  </action>
  <action name="m_editText">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/editTextState.png</normaloff>:/Resource/editTextState.png</iconset>
   </property>
   <property name="text">
    <string>编辑文字标签</string>
   </property>
  </action>
  <action name="m_editMile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/flag.png</normaloff>:/Resource/flag.png</iconset>
   </property>
   <property name="text">
    <string>编辑里程</string>
   </property>
  </action>
  <action name="m_forbidEdit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/forbidEdit.png</normaloff>:/Resource/forbidEdit.png</iconset>
   </property>
   <property name="text">
    <string>修改模式</string>
   </property>
   <property name="toolTip">
    <string>修改模式</string>
   </property>
  </action>
  <action name="m_editLine">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/line.png</normaloff>:/Resource/line.png</iconset>
   </property>
   <property name="text">
    <string>编辑线条</string>
   </property>
  </action>
  <action name="m_saveTopImg">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/cut.png</normaloff>:/Resource/cut.png</iconset>
   </property>
   <property name="text">
    <string>保存前景图片</string>
   </property>
  </action>
  <action name="m_openFolder">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/open_dir.png</normaloff>:/Resource/open_dir.png</iconset>
   </property>
   <property name="text">
    <string>打开目录</string>
   </property>
   <property name="toolTip">
    <string>打开目录</string>
   </property>
  </action>
  <action name="m_handlerImag">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resource/processs.png</normaloff>:/Resource/processs.png</iconset>
   </property>
   <property name="text">
    <string>解析图片轮廓</string>
   </property>
   <property name="toolTip">
    <string>解析图片轮廓</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
